src/main/groovy/edu/oregonstate/mist/directoryapi/DirectoryEntityResource.groovy

```diff
@@ -64,6 +69,7 @@ class DirectoryEntityResource extends Resource {
                 )
                 responseBuilder = ok(resultObject)
             } catch (LdapException ldapException) {
+                logger.error("Ldap Exception thrown when getting by search query", ldapException)
```

Comment:
Should probably log all exceptions, not just LdapException, for consistency with the other apis.